ts-client/src/amm/curve/constant-product.ts

```diff
@@ -28,23 +28,23 @@ function ceilDiv(lhs: BN, rhs: BN) {
 
 export class ConstantProductSwap implements SwapCurve {
   constructor() {}
+
+  get isStablePool(): boolean {
```

Comment:
I think this should not be part of `SwapCurve` interface. In the future, we might introduce more curve

Reply:
ahh ok